app/src/main/java/com/example/myapplication/ManaPickerActivity.java

```diff
@@ -0,0 +1,65 @@
+package com.example.myapplication;
+
+import android.os.Bundle;
+import android.view.View;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.viewpager.widget.ViewPager;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ManaPickerActivity extends AppCompatActivity {
+
+    ViewPager viewPager;
+    ManaPickerAdapter adapter;
+    List<ManaListItem> models;
```

Comment:
did you mean manot?

Reply:
Models are the 3 cards for the pageViewer(image,price,name), didn't want to make a confusion because we already use "manot" in another activity. 
we'll discuss this after we'll merge all the activities.